src/Symfony/Bundle/FrameworkBundle/Validator/ConstraintValidatorFactory.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Validator;
 
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Psr\Container\ContainerInterface;
```

Comment:
I think it's acceptable, the methods defined are the same and it will only fail if someone checks its interface which is unlikely.